docs/examples/reading-data.ipynb

```diff
@@ -0,0 +1,764 @@
+       "\n",
+       "  <!-- Colored Rectangle -->\n",
+       "  <polygon points=\"24.9485979497544,14.948597949754403 144.9485979497544,14.948597949754403 144.9485979497544,40.36121446433689 24.9485979497544,40.36121446433689\" style=\"fill:#ECB172A0;stroke-width:0\"/>\n",
+       "\n",
+       "  <!-- Text -->\n",
+       "  <text x=\"84.948598\" y=\"60.361214\" font-size=\"1.0rem\" font-weight=\"100\" text-anchor=\"middle\" >6000</text>\n",
+       "  <text x=\"164.948598\" y=\"27.654906\" font-size=\"1.0rem\" font-weight=\"100\" text-anchor=\"middle\" transform=\"rotate(0,164.948598,27.654906)\">20</text>\n",
+       "  <text x=\"7.474299\" y=\"52.886915\" font-size=\"1.0rem\" font-weight=\"100\" text-anchor=\"middle\" transform=\"rotate(45,7.474299,52.886915)\">1</text>\n",
+       "</svg>\n",
+       "        </td>\n",
+       "    </tr>\n",
+       "</table></div></li></ul></div></li><li class='xr-section-item'><input id='section-7d252b3f-7368-4ca0-9199-781f6e81cae8' class='xr-section-summary-in' type='checkbox' disabled ><label for='section-7d252b3f-7368-4ca0-9199-781f6e81cae8' class='xr-section-summary'  title='Expand/collapse section'>Attributes: <span>(0)</span></label><div class='xr-section-inline-details'></div><div class='xr-section-details'><dl class='xr-attrs'></dl></div></li></ul></div></div>"
+      ],
+      "text/plain": [
+       "<xarray.Dataset>\n",
+       "Dimensions:          (nMeshFaces: 3840, nFaceNodes: 4, nMeshNodes: 6000,\n",
+       "                      meshLayers: 20, time: 1)\n",
+       "Coordinates:\n",
+       "    mesh_node_x      (nMeshNodes) float64 ...\n",
+       "    mesh_node_y      (nMeshNodes) float64 ...\n",
+       "  * time             (time) float64 13.0\n",
+       "Dimensions without coordinates: nMeshFaces, nFaceNodes, nMeshNodes, meshLayers\n",
+       "Data variables:\n",
+       "    mesh             int32 ...\n",
+       "    mesh_face_nodes  (nMeshFaces, nFaceNodes) uint32 ...\n",
+       "    mesh_depth       (meshLayers, nMeshNodes) float64 ...\n",
+       "    v1               (time, meshLayers, nMeshNodes) float64 dask.array<chunksize=(1, 20, 6000), meta=np.ndarray>\n",
+       "    v2               (time, meshLayers, nMeshNodes) float64 dask.array<chunksize=(1, 20, 6000), meta=np.ndarray>\n",
+       "    v3               (time, meshLayers, nMeshNodes) float64 dask.array<chunksize=(1, 20, 6000), meta=np.ndarray>"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Opening 1 grid file\n",
+    "grid = ux.open_dataset(grid_path)\n",
+    "\n",
+    "# Opening 1 grid file and 1 data file\n",
+    "grid = ux.open_dataset(grid_path, data_paths[0])\n",
+    "\n",
+    "# Opening 1 grid file and 3 data files\n",
+    "grid = ux.open_dataset(grid_path, data_paths[0], data_paths[1], data_paths[2])\n",
+    "\n",
+    "# Accessing underlying dataset\n",
```

Comment:
Split the part that starts with this "Accessing underlying dataset" from its current cell into a new one. Create that new cell  in a way that you show the contents of both `grid` and `grid.ds`. It can be something like:

Let's have a look at the contents of our Grid object:
`grid`
Let's now look at the `xarray.dataset` that our Grid object wraps up:
`grid.ds`